File: source/real-order/src/main/java/edu/jmu/seven/sys/entity/Cuisine.java
package edu.jmu.seven.sys.entity;

import java.io.Serializable;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

/**
 * <p>
 * 
 * </p>
 *
 * @author Seven
 * @since 2021-07-16
 */
@TableName("vocation.cuisine")
@ApiModel(value = "Cuisine对象", description = "")
public class Cuisine implements Serializable {


    private Integer code;

    private String name;


    public Integer getCode() {
        return code;
    }

    public void setCode(Integer code) {
        this.code = code;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return "Cuisine{" +
        "code=" + code +
        ", name=" + name +
        "}";
    }
}

File: source/real-order/src/main/kotlin/edu/jmu/seven/entity/Cuisine.kt
package edu.jmu.seven.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableField
import com.baomidou.mybatisplus.annotation.TableId
import com.baomidou.mybatisplus.annotation.TableName
import java.io.Serializable

/**
 * @author Seven
 * @since 2021-07-16
 */
@TableName("vocation.cuisine")
class Cuisine(

    @TableField("name")
    var name: String
) : Serializable {
    @TableId(type = IdType.AUTO )
    var code: Int? = null
    override fun toString(): String {
        return "Cuisine{" +
                "code=" + code +
                ", name=" + name +
                "}"
    }
}